docker/README.md

```diff
@@ -1,63 +1,306 @@
+as possible.
+
+In general, you should interact with the production deployment by
+using `sudo` to run commands as `morphodict`.
+
+Feel free to add yourself to the `morphodict` and `morphodict-run` groups
+to more easily poke at the production environment.
+
+### Data paths
+
+Production data is stored in `/data_local/application-data`, and mounted
+into the containers. Data files should be all be readable, and some should
+also be writable, by the `morphodict-run` group.
+
+To help ensure that directories are usable by the `morphodict-run` group
+even if the `morphodict` user or someone else creates them, the base
+application directories are created with the `g+s` aka `0o2000` chmod bit.
+According to [inode(7)],
+
+> The set-group-ID bit (S_ISGID) has several special uses.  For a
+> directory, it indicates that … for that directory: files created there
+> inherit their group ID from the directory, not from the effective group
+> ID of the creating process, and directories created there will also get
+> the S_ISGID bit set.
+
+[inode(7)]: https://man7.org/linux/man-pages/man7/inode.7.html
+
+This means that files and directories created by the `morphodict` user
+inside the base directory will automatically inherit the `morphodict-run`
+group of their parent directories. However, do note that the `g+s` bit is
+silently removed when running chown, even if the user and group are the
+same before and after.
+
+### The helper tool
+
+Rather than giving you a bunch of commands to copy and substitute values
+into and paste into your terminal, morphodict gives you `docker/helper.py`,
+a tool that encapsulates all the fiddly commands you’d otherwise have to
+copy-and-paste (or memorize!), along with automatically filling in needed
+parameters from a single place: [docker/helper/settings.py][].
+
+Run `./docker/helper.py --help` to see more about what it can do.
+
+### Installation
+
+These steps should only be needed if you are setting up morphodict on a new
+production host. But if something goes wrong, it’s generally safe to repeat
+the steps to make sure everything is set up correctly.
+
+#### Privileged setup
+
+To set up initial users, groups, and base directories, all of which require
+root access, there is an ansible script in `docker/plays`.
+
+ It only needs to be run:
+  - When first setting up a new production morphodict host
+  - When adding a new dictionary
+  - If it is edited to effect some desired configuration change
+
+It’s off in its own little world because it requires sudo access to run,
+and it creates the users and directories that would normally have a
+checkout containing the setup code. When getting started, you can copy just
+this file plus `vars.yml` into any directory, and you’ll be able to run it
+from there.
+
+To install ansible, run this, **as your login user, not as root**:
+
+    pip3 install --user ansible
+
+To run the playbook in a mode where it will show what it will do:
```

Comment:
Maybe specify that this is a preview of the changes that will be made. I didn't fully understand what this meant until we ran it together.